packages/flask/src/UICarouselView/CarouselView/UICarouselViewContainer.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import type { UICarouselViewContainerProps, UICarouselViewPageProps } from '../types';
+// @ts-ignore
+// eslint-disable-next-line import/no-unresolved, import/extensions
+import { CarouselViewContainer } from './CarouselContainer';
+import { usePages } from './usePages';
+import { Pagination } from './Pagination';
+
+export const UICarouselViewContainer: React.FC<UICarouselViewContainerProps> = ({
+    testID,
+    children,
+    initialIndex = 0,
+    onPageIndexChange,
+    shouldPageMoveOnPress,
+    showPagination = true,
+}: UICarouselViewContainerProps) => {
```

Comment:
as @savelichalex noted one day that it's better to use regular `function name(...args)` for components function to avoid anonymous functions in React DevTools.

Plus, there is another concern about it: https://twitter.com/heroicamit/status/1437677666929315840?s=12

Can you change it please?